content/doc/book/system-administration/reverse-proxy-configuration-apache.adoc

```diff
@@ -75,9 +75,15 @@ http://localhost:80/jenkins). +
 Set the context path in Windows by modifying the `+jenkins.xml+`
 configuration file and adding --prefix=/jenkins (or similar) to the
 <arguments> entry. +
-Set the context path  when using the Ubuntu package by adding
-`+--prefix=/jenkins+` to `+JENKINS_ARGS+` in `+/etc/default/jenkins+` (
-or in `+/etc/sysconfig/jenkins+` for RHEL/CentOS package)
+Set the context path when using the Linux package
+by adding the following
+to `/etc/systemd/system/jenkins.service.d/override.conf`:
```

Comment:
Also refer to `systemd edit jenkins` here?
```suggestion
to `/etc/systemd/system/jenkins.service.d/override.conf` (for example, using `systemd edit jenkins`):
```

Reply:
Done in commit 4b280c8.